### PR TITLE
Update Service Plan

### PR DESCRIPTION
**Pull request by antechrestos**
This change adds the api to update a service plan (```PUT /v2/service_plans/:guid```)
See [this story](https://www.pivotaltracker.com/projects/816799/stories/114818681)

### CODE REVIEW

**Comment by nebhale** [comment]
@antechrestos Did you mean to close this and remove it?  Ah, because @Zteve beat you to it :)

File: cloudfoundry-client/src/main/lombok/org/cloudfoundry/client/v2/serviceplans/UpdateServicePlanRequest.java [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    private final String servicePlanId;
+
+    /**
+     * The visible flag
+     *
+     * @param public the visible flag
+     * @return the visible flag
+     */
+    @Getter(onMethod = @__(@JsonProperty("public")))
+    private final Boolean visible;
```
Comment (nebhale) [inline cloudfoundry-client/src/main/lombok/org/cloudfoundry/client/v2/serviceplans/UpdateServicePlanRequest.java]: @Zteve interesting that both you and @antechrestos chose the same name here.  Great minds...

Comment (Zteve) [inline cloudfoundry-client/src/main/lombok/org/cloudfoundry/client/v2/serviceplans/UpdateServicePlanRequest.java]: Not really. There is a precedent in the code already.

Comment (nebhale) [inline cloudfoundry-client/src/main/lombok/org/cloudfoundry/client/v2/serviceplans/UpdateServicePlanRequest.java]: For `public` -> `visible`?  Can you you please point me at that.  I wonder if it has slightly different semantics or if there's an improvement to be made.

Comment (nebhale) [inline cloudfoundry-client/src/main/lombok/org/cloudfoundry/client/v2/serviceplans/UpdateServicePlanRequest.java]: Ah, found it.  Yeah probably wrong there as well.

**Pull request closed without merging**